src/main/steps/common/common.content.ts

```diff
@@ -449,25 +459,39 @@ export const generateCommonContent = ({
   const referenceNumber = formattedCaseId(userCase?.id);
 
   const hasServiceApplicationInProgress = !!userCase?.receivedServiceApplicationDate;
-  const serviceApplicationType =
-    commonTranslations.generalApplication[
-      userCase?.applicant1InterimApplicationType === InterimApplicationType.SEARCH_GOV_RECORDS
-        ? 'searchGovRecords'
-        : (userCase?.alternativeServiceType as string)
-    ];
```

Comment:
I had to add this for search gov records for application submitted and hub page. Are we getting the text from a different source now?